pkg/apis/operator/v1alpha1/tektonconfig_lifecycle.go

```diff
@@ -21,107 +21,128 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var (
-	_ TektonComponentStatus = (*TektonConfigStatus)(nil)
+const (
+	ComponentsReady apis.ConditionType = "ComponentsReady"
+)
 
+var (
 	configCondSet = apis.NewLivingConditionSet(
-		DependenciesInstalled,
-		DeploymentsAvailable,
-		InstallSucceeded,
```

Comment:
shall we retain `InstallSucceeded` as the final condition. Then we could share that as the standard way to check whether all aspects of TektonConfig installation was completed successfully.

Reply:
currently i haven't added any cond for pruner .,. so kept a common `componentready` for all ..
we can look into it more
what would `InstallSucceeded` say?

Reply:
`InstallSucceeded` shows the success of Operator installation basically success of TektonConfig installation

Reply:
> currently i haven't added any cond for pruner .,. so kept a common componentready for all ..

ok.. copy that.
i recommend adding a final `InstallSucceed` so that QE and admins can use scripts for monitoring this one condition, no matter what how we add other intermediate conditions (or add more checks before setting `InstallSucceed: True`)

Reply:
by default `Ready` condition is added by knative/pkg which gets true only when all other cond get true
so `InstallSucceeded` is diff from `Ready` ?

Reply:
Oh.. great 👍 forgot about that.
That is enough.